README.md

```diff
@@ -1 +1,31 @@
-## RPi Control Center
+# RPi Control Center
+
+A control center application for Raspberry Pi, designed to manage and monitor various Raspberry Pi functionalities through a user-friendly interface.
+
+<img src="screenshots/en/img.png" width="75%" title="rpicc Activity">
+
```

Comment:
Can you please add these 2 as well
https://drive.google.com/file/d/1TvzNgIhXcxYd8-CjNZQF2rJ893CpO6Cr/view?usp=sharing
https://drive.google.com/file/d/1Jb5YMoe_ExkrR22nJTc2i_GYF2srHRkS/view?usp=sharing

Reply:
Ok Will add them Thanks!!